tests/test_packages.py

```diff
@@ -115,17 +169,37 @@ def test_positive_lock_package_versions(setup_install_pkgs, ansible_module):
     for result in contacted.values():
         logger.info(result['stdout'])
         assert 'Packages are locked.' in result['stdout']
+        assert 'Automatic locking of package versions is enabled in installer.' in result['stdout']
         assert "FAIL" not in result['stdout']
         assert result["rc"] == 0
     contacted = ansible_module.command(Packages.is_locked())
     for result in contacted.values():
         logger.info(result['stdout'])
-        assert 'Foreman related packages are locked' in result['stdout']
+        assert 'Packages are locked' in result['stdout']
         assert result["rc"] == 0
-    contacted = ansible_module.command("grep 'satellite' /etc/yum/pluginconf.d/versionlock.list")
-    for result in contacted.values():
-        logger.info(result['stdout'])
-        assert 'satellite' in result['stdout']
+    contacted = ansible_module.yum(
+        name='zsh',
+        state='absent'
+    )
+    for result in contacted.values():
+        assert result['rc'] == 0
+    contacted = ansible_module.command('yum install -y zsh')
+    for result in contacted.values():
+        assert result['rc'] == 1
+        assert "Use foreman-maintain packages install/update <package>" in result['stdout']
+    # Test whether FM packages install/ update command works as expected.
+    contacted = Connection(SAT_HOSTNAME, 'root').run(
```

Comment:
Create a common function where I will import it and use it
```
ex: 
from common_function import run

run(Packages.install())
```

Reply:
Added it in testfm.helpers